cdap-api/src/main/java/co/cask/cdap/api/artifact/ArtifactId.java

```diff
@@ -49,11 +49,7 @@ public ArtifactScope getScope() {
 
   @Override
   public String toString() {
-    return "ArtifactId{" +
-      "name='" + name + '\'' +
-      ", version=" + version +
-      ", scope='" + scope + '\'' +
-      '}';
+    return String.format("%s-%s-%s", scope, name, version);
```

Comment:
Forgot that this class is in fact in cdap-api. Will move the method into app-fabric.